server/src/utils/fight/fightMethods.ts

```diff
@@ -1740,6 +1737,14 @@ export const playFighterTurn = (
 
       const thrownWeapon = fighter.activeWeapon;
 
+      // Add throw step
+      fightData.steps.push({
+        a: StepType.Throw,
+        f: fighter.index,
+        t: opponent.index,
+        w: WeaponByName[fighter.activeWeapon.name],
+        k: keepWeapon ? 1 : 0,
+      });
```

Comment:
Why this change?

Reply:
The throw step is a new step that wasn't there when I started the migration , it was using id when you implemented it I had to fix it